ide-migration/server/migration/api/migration-service.js

```diff
@@ -45,30 +57,104 @@ class MigrationService {
         }
         workspace = workspaceManager.getWorkspace(workspaceName);
 
+        const deployables = [];
+
         for (let i = 0; i < lists.length; i++) {
             const file = lists[i];
             // each file's package id is based on its directory
             // if we do not get only the first part of the package id, we would have several XSK projects created for directories in the same XS app
-            const filePackageId = file.packageId.split('.')[0]; 
+            const projectName = file.packageId.split('.')[0];
 
-            let project = workspace.getProject(filePackageId)
+            let project = workspace.getProject(projectName)
             if (!project) {
-                workspace.createProject(filePackageId)
-                project = workspace.getProject(filePackageId)
+                workspace.createProject(projectName)
+                project = workspace.getProject(projectName)
+            }
+
+            if (!deployables.find(x => x.projectName === projectName)) {
+                deployables.push({
+                    project: project,
+                    projectName: projectName,
+                    artifacts: []
+                });
             }
 
             let fileRunLocation = file.RunLocation;
 
-            if (fileRunLocation.startsWith("/" + filePackageId)) { 
+            if (fileRunLocation.startsWith("/" + projectName)) {
                 // remove package id from file location in order to remove XSK project and folder nesting
-                fileRunLocation = fileRunLocation.slice(filePackageId.length + 1);
+                fileRunLocation = fileRunLocation.slice(projectName.length + 1);
             }
 
             let projectFile = project.createFile(fileRunLocation);
             projectFile.setContent(file._content);
+
+            if (fileRunLocation.endsWith('hdbcalculationview')
+                || fileRunLocation.endsWith('calculationview')
+                || fileRunLocation.endsWith('hdbtable')) {
```

Comment:
For the time being we should put into HDI container only the **calculationview** and **hdbcalculationview**.